skyportal/models.py

```diff
@@ -3120,6 +3126,327 @@ class SourceNotification(Base):
     level = sa.Column(sa.String(), nullable=False)
 
 
+class GcnEvent(Base):
+    """Event information, including an event ID, mission, and time of the event."""
+
+    id = sa.Column(sa.Integer, primary_key=True, autoincrement=True)
+
+    dateobs = sa.Column(sa.DateTime, doc='Event time', unique=True, nullable=False)
+
+    gcn_notices = relationship(lambda: GcnNotice, order_by=lambda: GcnNotice.date)
```

Comment:
Mainly for consistency with the rest of SkyPortal, I would prefer to see the lambda functions in this class turned into normal arguments, i.e. `relationship('GcnNotice', order_by=GcnNotice.date)`. To have that work you probably have to define the `GcnEvent` class after the other data tables it depends on but that's perfectly fine.

Reply:
I am a little confused how to do this sa.ForeignKey(GcnEvent.dateobs) if we have now reordered these? putting it as a string leads to errors like: Ensure that referencing columns are associated with a ForeignKey or ForeignKeyConstraint, or specify a 'primaryjoin' expression.

Reply:
Looks like strings it is.